lms/envs/common.py

```diff
@@ -4190,8 +4190,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     },
     'twitter': {
         'display_name': 'Twitter',
```

Comment:
@dyudyunov @Inferato I think so, yep.. and there's a couple more references that could be fixed here too:

* 1 mention in [cms.envs.common](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/cms/envs/common.py#L171)
* 1 enum label in [certificates.models](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/lms/djangoapps/certificates/models.py#L47)
* 2 mentions in [certificates/_accomplishment-banner.html](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/lms/templates/certificates/_accomplishment-banner.html#L60)
* [student/tests/test_views.py](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/common/djangoapps/student/tests/test_views.py#L314) and whatever templates are driving this test.

And maybe [more](https://github.com/search?q=repo%3Aopenedx%2Fedx-platform+twitter&type=code)? Mentions made in `js/fixtures` or `js/spec` can be ignored I think (e.g. [1](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/xmodule/js/fixtures/video_all.html#L46), [2](https://github.com/openedx/edx-platform/blob/bca34c0993aa7004b86cbf318ea55fd3a0ca5d95/lms/static/js/fixtures/course_sharing/course_listings.html#L35-L37)), since they're only used in testing.